src/main/java/com/github/vkorobkov/jfixtures/sql/SqlBase.java

```diff
@@ -15,8 +17,14 @@
     @SneakyThrows
     @Override
     default void cleanTable(Appender appender, CleanTable cleanTable) {
-        String table = escapeTableOrColumn(cleanTable.getTable());
-        appender.append("DELETE FROM ", table, ";\n");
+        val table = escapeTableOrColumn(cleanTable.getTable());
+        val cleanMethod = cleanTable.getCleanMethod();
+
+        if (CleanMethod.DELETE == cleanMethod) {
```

Comment:
Expand to switch/case with throwing `IllegalStateException` in `default` branch?